src/GraphQL/Utilities/SchemaPrinter.cs

```diff
@@ -322,9 +329,36 @@ public string FormatDefaultValue(object value)
                 return value.ToString().ToLower();
             }
 
+            if (IsEnumType(graphType))
+            {
+                return "{0}".ToFormat(SerializeEnumValue(graphType, value));
+            }
+
             return "{0}".ToFormat(value);
         }
 
+        private static bool IsEnumType(IGraphType type)
+        {
+            if (type is NonNullGraphType)
+            {
+                var nullable = (NonNullGraphType)type;
+                type = nullable.ResolvedType;
+            }
+
+            return type is EnumerationGraphType;
+        }
```

Comment:
There is a `GraphQLExtensions.GetNamedType` that can do this unwrapping for you (and it includes multiple levels).  I think that would simplify this to:

```
private static bool IsEnumType(IGraphType type)
{
    return type.GetNamedType() is EnumerationGraphType;
}
```

Reply:
Confirmed working - will update.